services/venice-router/src/main/java/com/linkedin/venice/router/RouterServer.java

```diff
@@ -629,25 +630,14 @@ public AsyncFuture<LongSupplier> getLongTailRetryMilliseconds(
     final SslInitializer sslInitializer;
     if (sslFactory.isPresent()) {
       sslInitializer = new SslInitializer(SslUtils.toAlpiniSSLFactory(sslFactory.get()), false);
-      if (config.isThrottleClientSslHandshakesEnabled()) {
-        ExecutorService sslHandshakeExecutor = registry.factory(ShutdownableExecutors.class)
-            .newFixedThreadPool(
-                config.getClientSslHandshakeThreads(),
-                new DefaultThreadFactory("RouterSSLHandshakeThread", true, Thread.NORM_PRIORITY));
-        int clientSslHandshakeThreads = config.getClientSslHandshakeThreads();
-        int maxConcurrentClientSslHandshakes = config.getMaxConcurrentClientSslHandshakes();
-        int clientSslHandshakeAttempts = config.getClientSslHandshakeAttempts();
-        long clientSslHandshakeBackoffMs = config.getClientSslHandshakeBackoffMs();
-        if (useEpoll) {
-          sslResolverEventLoopGroup = new EpollEventLoopGroup(clientSslHandshakeThreads, sslHandshakeExecutor);
-        } else {
-          sslResolverEventLoopGroup = new NioEventLoopGroup(clientSslHandshakeThreads, sslHandshakeExecutor);
-        }
-        sslInitializer.enableResolveBeforeSSL(
```

Comment:
I assume this thread pool is used for DNS resolution, not for every phase of ssl handshake.
So we are confident that this feature is not required any more, right?

Reply:
Yeah. This was never used. We tried enabling it once, but we could never roll it out